### PR TITLE
Add Consoles and Time extension

### PR DESCRIPTION
**Pull request by SIPC**
Move extensions to the same folder

### CODE REVIEW

**Comment by OeildeLynx31** [comment]
Hello, I also created a navigator console extension ([source code](https://oeildelynx.top7box.com/fichiers/extension/console.js) | [preview](https://turbowarp.org/editor?extension=https://oeildelynx.top7box.com/fichiers/extension/console.js)). It is my first extension. Is it good? I will add the "time" function. If you want, you can copy blocks ... And do you have advices for me?

**Comment by SIPC** [comment]
> Hello, I also created a navigator console extension ([source code](https://oeildelynx.top7box.com/fichiers/extension/console.js) | [preview](https://turbowarp.org/editor?extension=https://oeildelynx.top7box.com/fichiers/extension/console.js)). It is my first extension. Is it good? I will add the "time" function. If you want, you can copy blocks ... And do you have advices for me?

This is good, I forgot about console.group :(

**Comment by SIPC** [comment]
Can the extension be i18n?

File: extensions/-SIPC-/Time.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    const icon2 = 'data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHdpZHRoPSIxNzUiIGhlaWdodD0iMTc1IiB2aWV3Qm94PSIwLDAsMTc1LDE3NSI+PGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoLTE1Mi41LC05Mi41KSI+PGcgZGF0YS1wYXBlci1kYXRhPSJ7JnF1b3Q7aXNQYWludGluZ0xheWVyJnF1b3Q7OnRydWV9IiBmaWxsPSIjZmZmZmZmIiBmaWxsLXJ1bGU9Im5vbnplcm8iIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLXdpZHRoPSIxIiBzdHJva2UtbGluZWNhcD0iYnV0dCIgc3Ryb2tlLWxpbmVqb2luPSJtaXRlciIgc3Ryb2tlLW1pdGVybGltaXQ9IjEwIiBzdHJva2UtZGFzaGFycmF5PSIiIHN0cm9rZS1kYXNob2Zmc2V0PSIwIiBzdHlsZT0ibWl4LWJsZW5kLW1vZGU6IG5vcm1hbCI+PHBhdGggZD0iTTMyNy41LDE4MGMwLDQ4LjI4MTI1IC0zOS4yMTg3NSw4Ny41IC04Ny41LDg3LjVjLTQ4LjI4MTI1LDAgLTg3LjUsLTM5LjIxODc1IC04Ny41LC04Ny41YzAsLTQ4LjI4MTI1IDM5LjIxODc1LC04Ny41IDg3LjUsLTg3LjVjNDguMjgxMjUsMCA4Ny41LDM5LjIxODc1IDg3LjUsODcuNXpNMjgzLjU5Mzc1LDE5OS42ODc1bC0zNy4zNDM3NSwtMjEuNTYyNXYtNDguMTI1aC0xMi41djU2LjI1aDEuNzE4NzVsNDEuODc1LDI0LjIxODc1eiIvPjwvZz48L2c+PC9zdmc+PCEtLXJvdGF0aW9uQ2VudGVyOjg3LjU6ODcuNS0tPg==';
+    class Time {
+        constructor () {}
+        getInfo() {
+            return {
+                id: 'Time',
```
Comment (GarboMuffin) [inline extensions/-SIPC-/Time.js:9]: change the IDs to something like "sipctime" "sipcconsole"

**Comment by GarboMuffin** [comment]
let's put encoding.js back in its old spot. renaming the files breaks projects so that can be dealt with later

File: extensions/-SIPC-/Time.js (hunk 1 above)
Comment (SIPC) [reply extensions/-SIPC-/Time.js:9]: OK

**Comment by SIPC** [comment]
> let's put encoding.js back in its old spot. renaming the files breaks projects so that can be dealt with later

OK!

**Comment by SIPC** [comment]
Extended images have been sent to [](https://github.com/TurboWarp/extensions/issues/90)

**Pull request merged**